packages/multichain-network-controller/src/constants.ts

```diff
@@ -0,0 +1,45 @@
+import { NetworkStatus } from '@metamask/network-controller';
+
+import type {
+  MultichainNetworkConfiguration,
+  MultichainNetworkMetadata,
+} from './MultichainNetworkController';
+
+export const bitcoinCaip2ChainId = 'bip122:000000000019d6689c085ae165831e93';
+export const solanaCaip2ChainId = 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp';
```

Comment:
Those are now exported by the `keyring-api`, but not sure we want to have an extra deps to this package here?

We keep repeating those constants here and there, so for now we could just depend on it until we move them somewhere else if need be, see:
- https://github.com/MetaMask/accounts/blob/v24.0.0/packages/keyring-api/src/btc/constants.ts#L7-L11
- https://github.com/MetaMask/accounts/blob/v24.0.0/packages/keyring-api/src/sol/constants.ts#L7-L9